### PR TITLE
Logo Designs

### PR DESCRIPTION
**Pull request by jbeguna04**
(Put an `X` inside the `[ ]` to denote check mark `[X]`.)

- [x] **If creating a new file :**
  - [x] added links to it in the README files ?
  - [ ] included tests with it ?
  - [x] added description (overview of algorithm, time and space compleixty, and possible edge case) in docstrings ?

- [ ] **if done some changes :**
  - [ ] wrote short description in the PR explaining what the changes do ?
  - [ ] Fixes #[issue number] if related to any issue

- [ ] **other**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<p align="center"><img src="https://raw.githubusercontent.com/jbeguna04/algorithms/master/LogoDesins/128pxblue.png" /></p>
 <p align="center"><img width="40%" src="docs/source/_static/algorithms_logo.png" /></p>
```
Comment (goswami-rahul) [inline README.md]: Please also replace the typography too, with the one that was finalized in #369. Thanks.

**Comment by jbeguna04** [comment]
Already changed the typography in the ReadMe File. Thank You for the review

**Comment by keon** [comment]
sleek

**Comment by jbeguna04** [comment]
Very honored to work with you all. Til next time :)

**Pull request merged**